templates/zerver/api/update-user.md

```diff
@@ -0,0 +1,34 @@
+# Update User
+
+Update the data of any user in the realm.
+
+`PATCH {{ api_url }}/v1/users/{user_id}`
+
+## Usage examples
+
+{start_tabs}
+{tab|python}
+
+{generate_code_example(python)|/users/{user_id}:patch|example}
+
+{tab|curl}
+
+{generate_code_example(curl)|/users/{user_id}:patch|example}
+
+{end_tabs}
+
+## Arguments
+
+{generate_api_arguments_table|zulip.yaml|/users/{user_id}:patch}
+
+## Response
+
+#### Example response
+
+A typical successful JSON response may look like:
+
+{generate_code_example|/users/{user_id}:patch|fixture(200)}
+
+A typical unsuccessful JSON response :
```

Comment:
The space before the `:` looks bad, please fix this (and self-review your PRs for this sort of type!)